.changeset/quick-peas-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/sdk": major
```

Comment:
```suggestion
"@thirdweb-dev/sdk": patch
```
this should not be `major` version but can be done under `patch` since it has no breaking changes or big new features